Security/PasswordResetToken.php

```diff
@@ -58,4 +58,16 @@ public static function generate()
 
         return new self($token);
     }
+
+    /**
+     * Check if a token is equal to a different token.
+     *
+     * @param PasswordResetToken $token
+     *
+     * @return bool
+     */
+    public function equals(PasswordResetToken $token)
+    {
+        return $token->getToken() === $this->token;
```

Comment:
protip: je kan van hieruit dit checken zonder de getter (`$token->token === $this->token`). Dan ben je 100% zeker dat je de properties vergelijkt en dat de getter niets raars doet :)